src/main/java/org/generationcp/middleware/dao/germplasmlist/GermplasmListDAO.java

```diff
@@ -106,6 +107,24 @@
 			+ "      OR desig = :q OR listname = :q " + "      OR desig = :qNoSpaces "
 			+ "      OR desig = :qStandardized " + ")";
 
+	private static final String COPY_LISTDATA_TO_NEW_LIST = "INSERT INTO listdata (listid, gid, entryid, entrycd, source, desig, grpname, lrstatus, llrecid) "
```

Comment:
Move to queries into their respective DAO classes:
`GermplasmListDataDAO`, `GermplasmListDataViewDAO`, `getGermplasmListDataDetailDAO`